RalstoniaWiltDashboard/AboutPage.Rmd

```diff
@@ -1,13 +1,13 @@
 # About the *Ralstonia*  Wilt Dashboard
 
 Welcome to the *Ralstonia* Wilt Dashboard, a comprehensive tool designed to explore and analyze the genetic diversity of *Ralstonia solanacearum* species complex (RSSC).
+The *Ralstonia* Wilt Dashboard integrates data from hundreds of sources. Please read our [companion pre-print](https://www.biorxiv.org/content/10.1101/2020.07.13.189936) on BioRxiv for more details: https://www.biorxiv.org/content/10.1101/2020.07.13.189936
 
-Our purpose is to communicate public data to stakeholders through a user-friendly dashboard that visualizes: 
+## Purpose
 
-- Geographic distribution patterns of *Ralstonia solanacearum* species complex (RSSC) lineages
-- Patterns of *Ralstonia solanacearum* species complex (RSSC) host range 
-
-The *Ralstonia* Wilt Dashboard integrates data from hundreds of sources. Please read our companion pre-print on BioRxiv for more details: https://www.biorxiv.org/content/10.1101/2020.07.13.189936
+- Communicate public data to stakeholders through a dashboard that visualizes geographic distribution patterns of *Ralstonia solanacearum* species complex (RSSC) lineages and patterns of *Ralstonia solanacearum* species complex (RSSC) host range 
+- To build a framework for understanding how and where exotic RSSC lineages can enter, establish, and spread in the U.S. 
```

Comment:
```suggestion
- To provide a framework for understanding how and where exotic RSSC lineages can enter, establish, and spread in the U.S. 
```

As it's built already, do we want to change the wording to "provide"? Or leave this out?